cypress.config.js

```diff
@@ -4,7 +4,7 @@ module.exports = defineConfig({
 	e2e: {
 		baseUrl: 'https://localhost:3000',
 		pageLoadTimeout: 1200000,
-		video: true,
+		video: false,
```

Comment:
@Vadman97 have you ever used these? Didn't seem like they were used, so thought we could disable. Let me know what you think.